core/domain/feedback_services.py

```diff
@@ -819,6 +819,7 @@ def enqueue_feedback_message_batch_email_task(user_id):
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
 
 
+@transaction_services.run_in_transaction_wrapper
 def enqueue_feedback_message_instant_email_task(user_id, reference):
```

Comment:
As mentioned in the other PR, please add _transactional as a suffix to all methods with this decorator. I have also modified the instructions for #11431 accordingly.

Could you also please do these changes as part of #11778? I think it makes sense to have them all in one PR since they are related, no need to split them into two. (It ends up being twice the review + CI load otherwise.) Thanks!

Reply:
Sure @seanlip ,
So sorry for the inconvenience I'll just update everything and get back to you.
Thanks for your support.